frontend/src/metabase/visualizations/lib/scalar_utils.unit.spec.ts

```diff
@@ -88,5 +88,18 @@ describe("scalar utils", () => {
       expect(displayValue).toBe(fullScalarValue);
       expect(displayValue).toBe("10,010,010,010");
     });
+
+    it("displayValue is always compact when formatOptions.compact is true", () => {
+      const value = 10010010010;
+      const width = 350;
+
+      const { displayValue, fullScalarValue } = compactifyValue(value, width, {
+        ...formatOptions,
+        compact: true,
+      }) as { displayValue: string; fullScalarValue: string };
+
+      expect(displayValue).toBe("10.0B");
+      expect(fullScalarValue).toBe("10,010,010,010");
+    });
```

Comment:
I think this test case should be moved to `compute.unit.spec.js` since we are moving the compact setting to the `computeTrend` function - since we want that behavior to be replicated for free on static-viz.

Maybe we put this test in a new section: "format options"?

Reply:
Good call, added tests to `compute.unit.spec.js` in [`c6ec018`](https://github.com/metabase/metabase/pull/36992/commits/c6ec0187d5727e92b0dd8cc929d4498f5c6aeb40). How about we keep that `scalar_utils` test? Let's ensure `compactifyValue` respects the `compact` option